rust/protocol/src/storage/inmem.rs

```diff
@@ -25,6 +31,10 @@ pub struct InMemIdentityKeyStore {
 }
 
 impl InMemIdentityKeyStore {
+    /// Create a new instance.
+    ///
+    /// `key_pair` corresponds to [traits::IdentityKeyStore::get_identity_key_pair], and `id`
+    /// corresponds to [traits::IdentityKeyStore::get_local_registration_id].
     pub fn new(key_pair: IdentityKeyPair, id: u32) -> Self {
```

Comment:
You may even want to rename the argument `id` to `registration_id` for clarity.

Reply:
Done! Thanks!